engine/api/src/main/scala/pl/touk/nussknacker/engine/api/definition/ParameterEditor.scala

```diff
@@ -0,0 +1,76 @@
+package pl.touk.nussknacker.engine.api.definition
```

Comment:
It is API - are you sure that it won't break anything? Is there a need to write some migration guide?

Reply:
From what I know this API is consumed by ui client, which was migrated to use new API